src/main/java/frc/robot/subsystems/WheelSubsystem.java

```diff
@@ -78,19 +99,33 @@ public CANSparkMax getSpeedMotor(){
     public AnalogEncoder getEncoder(){
         return turningEncoder;
     }
-    public double getTargetVoltage() {
-        return targetVoltage;
+    public double gettargetSpeed() {
+        return targetSpeed;
     }
 
     public double getEncoderPosition() {
+        // return turningEncoder.getAbsolutePosition();
         double rawPos = turningEncoder.getAbsolutePosition() - turningEncoder.getPositionOffset();
         if (rawPos < 0)
-            return 1 + rawPos;
+            return -rawPos;
         else
-            return rawPos;
+            return 1 - rawPos;
```

Comment:
What's wrong with `turningEncoder.getPosition`?

Reply:
Interestingly, that is not an actual function! The documentation for getAbsolutePosition says 
```"getAbsolutePosition()` - getPositionOffset() will give an encoder absolute position relative to the last reset. This could potentially be negative, which needs to be accounted for."```